src/common/log.ts

```diff
@@ -1,15 +1,44 @@
 import * as riverpig from 'riverpig'
+import { LoggerConfig } from 'riverpig'
+import { Logger } from 'riverpig'
```

Comment:
Couldn't this be one statement, `import { LoggerConfig, Logger } from 'riverpig'` or am I missing something here?